package.json

```diff
@@ -59,6 +61,7 @@
     "checkRemoteHint": "Run \"git remote -v\" for a list of available remote repositories.\n",
     "publishHint": "Ensure that you have access to publish to the registry and that the current version does not already exist.\n",
     "publishDevHint": "Ensure that you have access to publish to the registry and that the current version does not already exist.\nIf the npm tag specified by --tag is recognizable as a valid semver version number, it will be rejected by npm. This is because version numbers and tags share a common namespace for npm packages.\n",
+    "checkFilesTrackingHint": "Either the file ${filesToClean} is not being tracked or does not exist.",
```

Comment:
Again i think this actually refers to a single file as opposed to many, and should be adjusted to reflect that. Also maybe reword to something like "${file} was not cleaned. It is either not being tracked or does not exist."